lizard.py

```diff
@@ -679,15 +679,26 @@ def print_warnings(option, scheme, warnings):
     warn_str = "!!!! Warnings ({0}) !!!!".format(
         ' or '.join("{0} > {1}".format(
             k, val) for k, val in option.thresholds.items()))
-    print("\n" + "=" * len(warn_str) + "\n" + warn_str)
-    print(scheme.function_info_head())
-    for warning in warnings:
+    for _, warning in enumerate(warnings):
+        if warning_count == 0:
+            print("\n" + "=" * len(warn_str) + "\n" + warn_str)
+            print(scheme.function_info_head())
         warning_count += 1
         warning_nloc += warning.nloc
         print(scheme.function_info(warning))
+    else:
+        if warning_count == 0:
```

Comment:
The tests are still passing locally without the else, so I'm gonna push the change and see if the Travis build will pass.